bash_completion

```diff
@@ -390,6 +390,52 @@ _comp_expand_glob()
     return 0
 }
 
+# Split a string and assign to an array.
```

Comment:
Is the rationale for this function mostly to avoid repeated manual management of `noglob` and `$IFS`, or is there something else to it? Perhaps add a few words of reasoning here why/when this should be used.

Reply:
Thank you. I have updated the comment.